source/Parser.cpp

```diff
@@ -1907,6 +1914,125 @@ ParameterDeclarationSyntax* Parser::parseParameterPort() {
     return alloc.emplace<ParameterDeclarationSyntax>(Token(), type, parseOneVariableDeclarator());
 }
 
+ClockingSkewSyntax* Parser::parseClockingSkew() {
+    Token edge, hash;
+    ExpressionSyntax* value = nullptr;
+    if (peek().kind == TokenKind::EdgeKeyword || peek().kind == TokenKind::PosEdgeKeyword || peek().kind == TokenKind::NegEdgeKeyword) {
```

Comment:
There's a peek() method that takes a TokenKind and returns a bool which would make this easier.